helpers/room_filter/room_filter.py

```diff
@@ -56,6 +56,11 @@ def location_join_resources():
     return location_query
 
 
+def location_join_office():
+    location_query = Location.query.join(Office)
+    return location_query
```

Comment:
Hello @kevinbett , I don't think we need this because we are not doing any joins here. In the `admin_roles.py` file just import the `Location Model` and we will be good to go.

Reply:
Hi @mutoromike I have made the changes requested above.